.github/workflows/release.yml

```diff
@@ -51,9 +51,7 @@ jobs:
       contents: write # For asset uploads.
     uses: slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@v1.9.0
     with:
-      # TODO(2680): re-enable go-version-file
-      # go-version-file: "go.mod"
-      go-version: '1.20'
+      go-version-file: "go.mod"
```

Comment:
I'm not sure what the issue was before, but using `go-version-file` instead of `go-version` lets use specify which version to use in only one place.
* https://github.com/actions/setup-go/blob/3d65fa57fcbfe4a359b6b71a6c65e6eec12984eb/README.md#getting-go-version-from-the-gomod-file

Reply:
Yeah, I remember we fixed an issue with it and I think that was in `1.9.0`.
https://github.com/slsa-framework/slsa-github-generator/issues/2661